test/index.js

```diff
@@ -45,18 +46,42 @@ const assert = require('assert')
   const cliTestsDir = path.join(__dirname, 'cli');
   const cliTests = await fs.readdir(cliTestsDir);
   suite('cli tests', async () => {
-    const supportCommand = '../../../bin/support';
-    cliTests.forEach((testDir) => {
+    const supportCommand = path.normalize('../../../bin/support');
+    cliTests.forEach((testDir, i) => {
       test(`cli test: ${testDir}`, () => {
         const cwd = path.join(__dirname, 'cli', testDir);
         const rawCommand = fs.readFileSync(path.join(cwd, 'command')).toString();
         const command = rawCommand.replace(/\$\{cwd\}/g, cwd);
         const expected = fs.readFileSync(path.join(cwd, 'expected')).toString();
-        const result = childProcess.execSync(`${supportCommand} ${command}`, { cwd: cwd }).toString();
+        // const expectedErr = fs.readFileSync(path.join(cwd, 'expected-errors')).toString(); // TODO
+        const expectedErr = 'TODO';
+
+        const stdOut = buildStringWriter({ objectMode: false });
+        const processOpts = {
+          cwd: cwd,
+          stdio: [0, 'pipe', stdOut]
+        };
+        const result = childProcess.execSync(`${supportCommand} ${command}`, processOpts).toString();
```

Comment:
```suggestion
        const result = String(childProcess.execSync(`${supportCommand} ${command}`, processOpts));
```

Reply:
@ljharb Could you please elaborate a bit more here with this string constructor suggestion? Is it some kind of ergonomic thing or there is something else?

Reply:
Calling .toString directly isn’t guaranteed to return a string; it’s better to always use `String()`.